src/editors/sharedComponents/ImageUploadModal/ImageSettingsModal/hooks.js

```diff
@@ -331,8 +331,12 @@ export const onSaveClick = ({
   })) {
     altText.error.dismiss();
     altText.validation.dismiss();
+    let altTextValue = altText.value;
+    if (altTextValue.includes('"')) {
```

Comment:
Why do you care to test with the if condition? What about just do replace() directly. It's no harm if no " character is in the text.

Reply:
Also, please add a developer comment on the impact of " character. It really breaks HTML, correct?

Reply:
I've tested without if condition. And yes, it is working perfectly without if condition. Thanks